docs/getting-started/user-guides/paymetheus.md

```diff
@@ -0,0 +1,60 @@
+# **Paymetheus** #
+
+Paymetheus is a simple to use graphical wallet for Decred. With it, you can send and receive DCR, 
+purchase tickets for [PoS voting](/mining/proof-of-stake.md), get a history of all your transactions and more.
+About the only thing Paymetheus doesn't do is PoS voting.
+
+---
+
+## **Download and Install** ##
+The first thing you'll need to do is get a copy of Paymetheus. You will find the downloads [here](https://github.com/decred/decred-binaries/releases). 
+Download the file for your operating system and install. Once you're done, run the 'Decred' program.
+You'll notice a program called "Paymetheus Standalone". This does not include the 'dcrd' program 
+required to connect to the Decred network, so don't use this one.
```

Comment:
I understand the audience, but I don't think they are all stupid people who can't learn.  There is a good reason why we ship with that shortcut and even if we don't go into detail about why we are not covering it here, we shouldn't downplay it as useless.